Makefile

```diff
@@ -194,6 +194,13 @@ antctl: $(ANTCTL_BINARIES)
 antctl-release:
 	@$(GO) build -o $(BINDIR)/$(ANTCTL_BINARY_NAME) $(GOFLAGS) -ldflags '-s -w $(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+.PHONY: check-copyright
+check-copyright: 
```

Comment:
Can you add some options to allow user to check the files with missing copyright only? and print a message in the final result how to rerun and add license.